frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDasboard.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
```

Comment:
Can you handle the logic here for
https://www.notion.so/TEC-Hour-based-event-credits-are-not-calculated-correctly-87db9eb80ec24f76a80a4262ecbe8033

and

https://www.notion.so/TEC-Implement-handling-of-duplicate-submissions-to-the-same-TEC-event-e1a8d38152314341805ef72129f4c12e?

It looks like you're passing in the `TeamEventAttendance` now. That's good. We would make a call to  `/getTeamEvent` and pass in the `eventUuid`. This will tell us the number of credits and whether or not the event has hours.

The `TeamEventCreditDashboard` component implementation needs to be reworked to take in the `TeamEventAttendance` arrays.

Reply:
<img width="676" alt="Screenshot 2023-04-13 at 12 20 16 AM" src="https://user-images.githubusercontent.com/70241445/231652545-a30cd6b6-0ff2-46fe-af07-4d6aae47cc7a.png">

This is what the member-side dashboard looks like with mock data. It accounts for hours attended calculated correctly and multiple submissions to the same TEC.